src/Uno.Templates/content/unoapp/GitHub/workflows/ci.yml

```diff
@@ -0,0 +1,84 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - $DefaultBranchName$
+      - release/**
+
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches:
+      - $DefaultBranchName$
+      - release/**
+env:
+  STEP_TIMEOUT: 60
+
+jobs:
+  smoke_test:
+    name: Smoke Test
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Dependencies
+        timeout-minutes: ${{ fromJSON(env.STEP_TIMEOUT) }}
+        uses: "./.github/workflows/steps/install_dependencies"
+        with:
+          dotnet-version: 'true'
+
+      # Setup the versioning for the app
+      - name: Setup GitVersion
+        uses: gittools/actions/gitversion/setup@v0.9.9
+        with:
+          versionSpec: '1.0.x'
+
+      - name: GitVersion
+        id: gitversion
+        uses: gittools/actions/gitversion/execute@v0.9.9
+        with:
+          useConfigFile: true
+          configFilePath: ./.github/workflows/config/gitversion.yml
+
+      # Add  MSBuild to the PATH: https://github.com/microsoft/setup-msbuild
+      - name: Setup MSBuild.exe
+        uses: microsoft/setup-msbuild@v1.0.2
+
+      - name: Build
+        shell: pwsh
+        run: msbuild ./MyExtensionsApp._1/MyExtensionsApp._1.csproj /r
+#if (useUnitTests)
+
+  unit_test:
+    name: Unit Tests
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup GitVersion
+        uses: gittools/actions/gitversion/setup@v0.9.9
+        with:
+          versionSpec: '5.x'
```

Comment:
versionSpec: '1.0.x' was used in previous job? Which version should we use?
Also update the setup version to match previous job ie v0.10.2